app/controllers/users_controller.rb

```diff
@@ -1,5 +1,22 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
   def show
     @user = current_user
```

Comment:
so since the items are being displayed from within the users show.html.erb we need to define @item and @items here in this method.